tracer/test/Datadog.Trace.Tests/Debugger/SymbolExtractor/Approvals/SymbolExtractorTest.Datadog.Trace.Tests.Datadog.Trace.Tests.Debugger.SymbolExtractorTest.verified.txt

```diff
@@ -0,0 +1,309 @@
+﻿{
+  "service": "test",
+  "env": "",
+  "version": "",
+  "language": "dotnet",
+  "scopes": [
+    {
+      "scopeType": "Assembly",
+      "name": "Datadog.Trace.Tests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=def86d061d0d2eeb",
+      "type": null,
+      "sourceFile": "C:\\dev\\Datadog\\dd-trace-dotnet\\tracer\\test\\Datadog.Trace.Tests\\bin\\Debug\\net6.0\\Datadog.Trace.Tests.dll",
+      "startLine": -1,
+      "endLine": 2147483647,
+      "languageSpecifics": null,
+      "symbols": null,
+      "scopes": [
+        {
+          "scopeType": "Class",
+          "name": "Datadog.Trace.Tests.Debugger.SymbolExtractorTest",
+          "type": null,
+          "sourceFile": "C:\\dev\\Datadog\\dd-trace-dotnet\\tracer\\test\\Datadog.Trace.Tests\\Debugger\\SymbolExtractorTest.cs",
+          "startLine": 24,
+          "endLine": 90,
+          "languageSpecifics": {
+            "accessModifiers": [
+              "Public"
+            ],
+            "annotations": [
+              "AutoLayout, AnsiClass, Class, Public, BeforeFieldInit"
+            ],
+            "superClasses": [
+              "System.Object"
+            ],
+            "interfaces": null,
+            "returnType": null
+          },
+          "symbols": [
+            {
+              "name": "_extractor",
+              "type": "SymbolExtractor",
+              "symbolType": "Field",
+              "line": -1
+            }
+          ],
+          "scopes": [
+            {
+              "scopeType": "Method",
+              "name": ".ctor",
```

Comment:
Do we need the `.`?

Reply:
That's a good point - the underline naming of constructors in .NET (regular c'tor is named `.ctor` and static c'tor is named `.cctor`), this fact might cause issues.
Maybe we should simply rename it to the Type name itself?

This point raises another question: how do we differentiate between different overloads? I can't see in the json the mention of 'signature'

Reply:
> Maybe we should simply rename it to the Type name itself?

But then, how do we differentiate between the constructor and the static constructor if we don't use the .NET convention you mentioned?

Reply:
We have `name`, `languageSpecifics.returnType` and `symbols` which contains args with types. That would be enough for the signature for a java method. Is it the same in dotnet?

Reply:
@ojung Sounds good.
@OmerRaviv Good point. Mentioning it in the language specific might help? (and then show it in a special way in the UI)
My concern is that the user will search for c'tor and he will see these `.ctor` methods and get confused

Reply:
@GreenMatan Sorry, I don't understand your point - why would the user search for `c'tor`?

Reply:
I don't think the user will search for `.ctor` but they will use the name of the c'tor itself but they will not find anything as the symbol is named `.ctor`. Rephrasing my concern: as the code is currently written, the user won't see c'tors in the way they might expect. They'll come across `ctor` (or `.cctor` for static constructors), which might confuse them.
I think we should be using the Type name (and rename `.ctor` and `.cctor`) and mention the fact that one method is the static constructor in the LanguageSpecific node.